### DISCUSSION TITLE
Manual Refit

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#154] [Ideas]
So, here's a suggestion, how about a feature where we drag 'n drop our ship PDFs on the site (or maybe something less complicated than that, like an accompanying code for the ship) and when we use it to allow us to edit our ship's attributes, departments and shields automatically? I say this because, sometimes in our sessions, we plan for a refit to take place, however since most of us can't afford (or aren't willing to pay, in my case) for Adobe Acrobat's premium version, I suggest a feature like this to be implemented into the site.

**Comment by Felderburg** [comment]
The tool should be exporting as fillable pdfs you can change the form fields in with adobe reader, not a kind you have to edit in full acrobat. I think.

Either way, there are form fillable pdfs available (for free from modiphius) that you could copy the data into.
